### ISSUE TITLE
APIs used for applications

### ISSUE THREAD

**Issue by ChunkyProgrammer** [IronMaltese/YouTube-Frontend-Comparison#9]
I think "Piped api support", "Invidious api support" and "local api" would be good options.

For example,
NewPipe uses a local api,
FreeTube uses a local api and Invidious,
Yattee uses Piped and Invidious,
LibreTube uses Piped

**Comment by davidcollini** [comment]
Good idea! I wanted to keep the chart very beginner-friendly so I stuck with visible features, but this is a great way to introduce people to backend information aswell
